pkg/bootstrap/config.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	"k8s.io/client-go/rest"
+)
+
+type Config struct {
+	// Namespace is the target namespace for Triggers resources
+	Namespace string
+
+	// Provider is the Git provider (github, gitlab, bitbucket)
+	Provider string
```

Comment:
does all these supported or only GitHub ??
If only GitHub can you add a comment saying today only support is for GitHub